lib/c-api/src/wasm_c_api/types/extern_.rs

```diff
@@ -21,6 +21,16 @@ pub unsafe extern "C" fn wasm_extern_type(e: &wasm_extern_t) -> Box<wasm_externt
 #[no_mangle]
 pub unsafe extern "C" fn wasm_externtype_delete(_et: Option<Box<wasm_externtype_t>>) {}
 
+#[no_mangle]
+pub unsafe extern "C" fn wasm_externtype_copy(
+    wasm_externtype: &wasm_externtype_t,
+) -> Box<wasm_externtype_t> {
+    // do shallow copy
```

Comment:
Is this really a shallow copy?  It makes a new box and clones the internal type

Reply:
It's not. Fixed!